src/impact-dashboard/MitigationInformation.tsx

```diff
@@ -97,13 +118,38 @@ const MitigationInformation: React.FC = () => {
 
   return (
     <Container>
+      <Description>
+        Select the most likely rate of spread. This will change the R0 (rate of
+        spread) in the model.
+      </Description>
+      <FormGrid>
+        <FormGridRow>
+          <FormGridCell>
+            <InputSelect
+              label="Rate of spread"
+              onChange={(e) =>
+                updateModel({
+                  rateOfSpreadFactor: e.target.value as RateOfSpread,
```

Comment:
This is a lot of nesting! You don't have to do it in this PR, but consider reducing nesting at some point. An easy way to do this is to pull out the InputSelect as follows:

```tsx
  let rateOfSpreadInput = <InputSelect ... />

  ...

  return (
    <Container>
      <Description>
        Select the most likely rate of spread. This will change the R0 (rate of
        spread) in the model.
      </Description>
      <FormGrid>
        <FormGridRow>
          <FormGridCell>
            {rateOfSpreadInput}
```